pkg/shell/base_index.js

```diff
@@ -479,7 +476,7 @@ function Router(index) {
                 return;
 
             /* Only control messages with a channel are forwardable */
-            } else if (control.channel === undefined && !forward_command) {
+            } else if (control.channel === undefined) {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18977-20230620-132626-075a58fe-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)